### PR TITLE
fix: unsignCookie return type (#93)

### PR DESCRIPTION
**Pull request by climba03003**
Resolve: #93 
Test cannot be added because there are no dependencies of ```fastify```, so the type cannot be tested

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: plugin.d.ts [hunk 1]
```diff
@@ -39,7 +39,11 @@ declare module 'fastify' {
      */
     unsignCookie(
       value: string,
-    ): string | false;
+    ): {
+      valid: boolean;
+      renew: boolean;
+      value: string | false;
```
Comment (salmanm) [inline plugin.d.ts:45]: `value` is either string or `null` in v5.
It's never `false`.

**Review by salmanm — CHANGES_REQUESTED** [review]

**Comment by mcollina** [comment]
Can you please add a unit test (https://github.com/fastify/fastify-cookie/blob/master/test/plugin.test-d.ts)?

File: plugin.d.ts (hunk 1 above)
Comment (climba03003) [reply plugin.d.ts:45]: thats true, missed the line 21

**Comment by climba03003** [comment]
> 
> 
> Can you please add a unit test (https://github.com/fastify/fastify-cookie/blob/master/test/plugin.test-d.ts)?

test for both cookie option and ```unsignCookie``` return are added..

**Review by salmanm — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Eomm — APPROVED** [review]

**Pull request merged**